test/utils/floating_point_utils.hpp

```diff
@@ -42,21 +53,46 @@ struct FP
     /// See: https://webassembly.github.io/spec/core/syntax/values.html#canonical-nan.
     static constexpr auto canon = UintType{1} << (num_mantissa_bits - 1);
 
-    T value{};
+private:
+    UintType m_storage{};  ///< Bits storage.
 
-    explicit FP(T v) noexcept : value{v} {}
+public:
+    FP() = default;
 
-    explicit FP(UintType u) noexcept : value{bit_cast<T>(u)} {}
+    FP(FloatType v) noexcept : m_storage{bit_cast<UintType>(v)} {}
```

Comment:
Does making it non-explicit help somewhere?

Reply:
Most of the arrays of test values are defined with it.